packages/components/psammead-radio-schedule/src/ProgramCard/__snapshots__/index.test.jsx.snap

```diff
@@ -226,34 +226,39 @@ exports[`ProgramCard should render correctly for live 1`] = `
         href="/news/articles/cn7k01xp8kxo"
       >
         <span
+          class=""
```

Comment:
Looks great, just wondering if it's fine to have empty classes in snapshots?

Reply:
I believe that since we don't add styles when declaring that styled component:
`const DurationTextWrapper = styled.span.attrs({ 'aria-hidden': 'true' })``;`
it leaves the class empty as it doesn't have to target it to apply styles. 

It also happens for the `DurationTextWrapper`.

Reply:
I also wondered why that was there...